test/e2e/tests/test_hpo.py

```diff
@@ -18,40 +18,47 @@
 import logging
 from typing import Dict
 
-from e2e import service_marker, create_sagemaker_resource
-from e2e.replacement_values import REPLACEMENT_VALUES
 from acktest.resources import random_suffix_name
 from acktest.k8s import resource as k8s
+from e2e import (
+    service_marker,
+    create_sagemaker_resource,
+    wait_for_status,
+)
+from e2e.replacement_values import REPLACEMENT_VALUES
+from e2e.bootstrap_resources import get_bootstrap_resources
+from e2e.common.config import *
```

Comment:
thanks!
it would be great if we can do `from e2e.common import config as cfg` instead of * because all are global variables

and then use variables like `cfg. JOB_STATUS_INPROGRESS`